src/basic/xattr-util.c

```diff
@@ -125,13 +124,13 @@ static int getxattrat_fake_prepare(
                 if (!(flags & AT_EMPTY_PATH))
                         return -EINVAL;
 
-                snprintf(ret_fn, STRLEN("/proc/self/fd/") + DECIMAL_STR_MAX(int) + 1, "/proc/self/fd/%i", dirfd);
+                format_proc_fd_path(ret_fn, dirfd);
```

Comment:
`dirfd` may be `AT_FDCWD`: 
`image_find()` -> `image_make()` -> `fd_getcrtime()/fd_getcrtime_at()` -> `fgetxattrat_fake()` -> `getxattrat_fake_prepare()`

(BTW, I noticed `path_getcrtime()` is not used anywhare.)

Reply:
Yes, this is consistent with the above backtrace.

Reply:
which is -100, so yep looks like it